charts/linkerd-control-plane/values.yaml

```diff
@@ -54,6 +54,8 @@ podAnnotations: {}
 podLabels: {}
 # -- Labels to apply to all resources
 commonLabels: {}
+# -- Annotations to apply to all resources
+commonAnnotations: {}
```

Comment:
You might consider using [`global:`](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values) for these, at least optionally with `| default` so that they can be set when the chart is used as a subchart.